rollup.config.js

```diff
@@ -12,6 +12,7 @@ export default {
       format: 'es'
     },
     {
+      name: 'ClipboardCopyElementMethods',
```

Comment:
In the case of no module loader, the exports from this module will be written to `window` under this key, e.g.:

```js
window.ClipboardCopyElementMethods.copyText('hello!')
```

I'm not happy with this name, butClipboardCopyElement is already taken as the class name. Should we attach these methods to that class instead of exporting them from the module? 🤔